resspect/tests/test_bazin.py

```diff
@@ -71,8 +76,9 @@ def test_fit_scipy():
     
     time = data['mjd'].values
     flux = data['flux'].values
+    fluxerr = data['fluxerr'].values
```

Comment:
change to fluxerr = 1 to fit in the unit test as it was done previously.

Reply:
this should solve the current unit test failure

Reply:
Okay, done. That solved it, thanks!